Remove the check for librt on Linux because as of autoconf 2.70 or 2.71
the check fails on macOS:
configure: error: Failed to find library with shm_open()
https://gitlab.gnome.org/GNOME/anjuta/-/issues/30
--- configure.ac.orig	2021-12-13 12:13:13.000000000 -0600
+++ configure.ac	2021-12-13 12:13:31.000000000 -0600
@@ -808,11 +808,6 @@
 
 AC_SUBST(SYMBOL_DB_SHM)
 
-dnl On Linux, need librt for shm_open/shm_unlink
-bck_LIBS="$LIBS"
-AC_SEARCH_LIBS(shm_open, rt, [SHM_LIBS="$LIBS"], [AC_MSG_ERROR([Failed to find library with shm_open()])])
-AC_SUBST(SHM_LIBS)
-LIBS="$bck_LIBS"
 
 dnl Test using autotest
 dnl -----------------------------
